File: 1897/solution.java
class Solution {
    public boolean makeEqual(String[] words) {
        int[] counts = new int[26];
        for (String word : words) {
            for (char c : word.toCharArray()) {
                counts[c - 'a']++;
            }
        }
        for (int num : counts) {
            if (num % words.length != 0) {
                return false;
            }
        }
        return true;
    }
}

File: 1897/solution.go
package main

func makeEqual(words []string) bool {
	counts := [26]int{}
	for _, word := range words {
		for _, char := range word {
			counts[int(char)-'a']++
		}
	}

	for _, num := range counts {
		if num%len(words) != 0 {
			return false
		}
	}
	return true
}